epochs/hippieeg/config/snakebids.yml

```diff
@@ -0,0 +1,124 @@
+bids_dir: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/epi_iEEG/ieeg'
+bids_dir_tsv: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/clinical_imaging/epi_iEEG/derivatives/seega_coordinates'
+output_dir: '/scratch/mcesped/epochs/results'
+
+#enable printing debug statements during parsing -- disable if generating dag visualization
+debug: False
+
+derivatives: False #will search in bids/derivatives if True; can also be path(s) to derivatives datasets
+
+#list of analysis levels in the bids app
+analysis_levels: &analysis_levels
+ - participant
+
+
+
+#mapping from analysis_level to set of target rules or files
+targets_by_analysis_level:
+  participant:
+    - ''  # if '', then the first rule is run
+
+#this configures the pybids grabber - create an entry for each type of input you want to grab
+# indexed by name of input
+#   dictionary for each input is passed directly to pybids get()
+#    https://bids-standard.github.io/pybids/generated/bids.layout.BIDSLayout.html#bids.layout.BIDSLayout.get
+
+pybids_inputs:
+  bold:
+    filters:
+      suffix: 'ieeg'
+      extension: '.edf'
+      datatype: 'ieeg'
+    wildcards:
+      - subject
+      - session
+      - acquisition
+      - task
+      - run
+
+pybids_inputs_annot:
+  bold:
+    filters:
+      suffix: 'annotations'
+      extension: '.tsv'
+      datatype: 'ieeg'
+    wildcards:
+      - subject
+      - session
+      - acquisition
+      - task
+      - run
+
+pybids_inputs_tsv:
+  bold:
+    filters:
+      suffix: 'native_SEEG'
+      extension: '.tsv'
+    wildcards:
+      - subject
+      - session
+      - acquisition
+      - task
```

Comment:
You could likely used the derivatives variable to define the path to this file which would likely look something like this `--derivatives /home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/clinical_imaging/epi_iEEG/derivatives/seega_coordinates/` in the commandline and bids should hopefully be able to fine the file based on the filters at that point.

Reply:
I used the custom path since the folder doesn't have the required json file to be interpreted as a bids folder but I'll try just in case. This is a temporal solution since I believe we should change the json files to be bids compliant